src/info.js

```diff
@@ -130,6 +130,81 @@ const parsePage = (data) => {
   return result
 }
 
+const parseReviews = (data) => {
+  const $ = cheerio.load(data)
+  var reviews = []
+
+  $('.borderDark').each(function (index) {
+    const review = {}
+    review.user = $(this).find('td > a').text().trim()
+    review.date = $(this).find('.mb8 div:nth-child(1)').text().trim()
+    review.read = $(this).find('.mb8 .spaceit').text().trim().replace('read', '')
+    var r = $(this).find('.borderClass').text().trim().split(' ').map(x => x.trim()).filter(x => x).map(x => isNaN(x) ? x : Number(x))
+    var rating = {}
+    rating[r[0]] = r[1]; rating[r[2]] = r[3]; rating[r[4]] = r[5]; rating[r[6]] = r[7]; rating[r[8]] = r[9]
+    review.rating = rating
+    review.helpful_for = Number($(this).find('.spaceit td .spaceit').text().trim().split(' ')[0])
+    review.text = $(this).find('.pt8').text().split('\n        \n      \n    \n\n                          \n    ')[1].split('Helpful\n  \n      \n      read more')[0].trim().replace('\n\n          \n        ', ' ')
```

Comment:
Is this really the only way? It will feel so messy to deal with in the future...